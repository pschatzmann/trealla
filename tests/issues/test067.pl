:- initialization(main).
:- use_module(library(lists)).

main :-
	prepare(List),
    writeln(List),
    sort(List, ListSorted),
    writeln(ListSorted).

prepare(List) :-
    append([A,B], [B,A], List).
