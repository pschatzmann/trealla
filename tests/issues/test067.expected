[_9,_10,_10,_9]
[_9,_10]
